doc/cla/individual/kurkop.md

```diff
@@ -8,4 +8,4 @@ declaration.
 
 Signed,
 
-Juan Pablo Arias jpa@devco.co https://github.com/kurkop
+Juan Pablo Arias j@kurlabs.com https://github.com/kurkop
```

Comment:
could you add a second line instead of replacing the previous one?

Reply:
@mart-e Yes. Done.